flamingpy/utils/viz.py

```diff
@@ -313,7 +320,7 @@ def _plot_EGraph_edges(ax, egraph, color_edges):
                 )
             edge_property = egraph.edges[edge].get(edge_attribute)
             color = color_dict.get(edge_property)
-        elif color_edges == True:
+        elif color_edges is True:
```

Comment:
Perhaps better to leave it as == True? This way color_edges can be set to, e.g., 0 or 1

Reply:
That's non-pythonic, later, we need to come up with a better solution. For this release, it makes sense to revert it.